doc/PACKAGERS.md

```diff
@@ -0,0 +1,83 @@
+Packaging
+=========
+
+Thank you for taking the time to package Rizin!
+
+There are some things to consider when packaging Rizin for your distribution.
+We will look at each step below.
+
+Cloning Source
+--------------
+
+Rizin uses git submodules to track versions of dependencies. You can check out
+the Rizin source and all dependencies using
+`git clone --recursive https://github.com/rizinorg/rizin`.
+
+If you change branches run `git submodule update --init --recursive` to ensure
+you are tracking the correct versions of the submodules.
+
+Building
+--------
+
+Rizin uses the `meson` build system. Command line flags passed to meson can
+change how Rizin is built.
+
+To define the base install location for Rizin use the `--prefix` flag when
+invoking `meson`. For system installs it is common to use `/usr`. If in doubt
+check your distributions packaging guidelines.
+
```

Comment:
I would add something like:
```
If you do not use `/usr` as a prefix, you may want to use `-Dlocal=disabled` to avoid `RPATH` in the installed binaries.
```